app/Site/Events/UserDeleted.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Site\Events;
+
+use App\Site\Models\User;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+class UserDeleted
+{
+    use Dispatchable;
+    use InteractsWithSockets;
```

Comment:
I'm still confused why we need `InteractsWithSockets`.

Reply:
Allows events to be pushed to browser clients via websockets. A websocket server will put into place with an upcoming PR.
Those traits are from Laravel's default event stubs.